### PR TITLE
[Feature #158124290] Add a persistent user registration route

### PR DESCRIPTION
**Pull request by migot01**
**What does this PR do?**
This PR will setup api endpoint for a persistent user registration 

How should this be manually tested?
Clone the branch into your local machine as follows
git clone https://github.com/migot01/Maintraq/tree/ft-user-registration-158124290git

The run the following to start the app:
export FLASK_APP=run.py
export FLASK_DEBUG=true
flask run

Go to postman select POST then write localhost:5000/api/v2/Auth/register
in the space provided ,input the required registration information
and click SENT

Any background context you want to provide?
None

What are the relevant pivotal tracker stories?
#158124290

### CODE REVIEW

File: app/models2.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import psycopg2
+
+GET_USERS = "SELECT * FROM users"
```
Comment (zmwas) [inline app/models2.py]: This query is not needed.

File: app/models2.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import psycopg2
+
+GET_USERS = "SELECT * FROM users"
+UPDATE_REQUESTS = ""
+FETCH_ALL_REQUESTS = "SELECT * FROM requests"
```
Comment (zmwas) [inline app/models2.py]: Add a WHERE clause for user_id e.g SELECT * FROM requests WHERE user_id = 1.

File: app/models2.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import psycopg2
+
+GET_USERS = "SELECT * FROM users"
+UPDATE_REQUESTS = ""
+FETCH_ALL_REQUESTS = "SELECT * FROM requests"
+GET_A_SINGLE_REQUEST = "SELECT * FROM requests where id='id'"
+GET_USERNAME= "SELECT username FROM users"
```
Comment (zmwas) [inline app/models2.py]: Add a WHERE clause

File: app/models2.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+def get_users(username):
+    #cur.execute(GET_USERS)
+   cur.execute(GET_USERS)
+   items = cur.fetchall()
+   conn.commit()
+   return items
+
+
+def create_user(user):
+   cur.execute("INSERT INTO USERS (first_name,last_name,email,username,password) values(%s,%s,%s,%s,%s)",(
```
Comment (zmwas) [inline app/models2.py]: Your query should have a WHERE clause.

Comment (migot01) [reply app/models2.py]: cool done

**Pull request merged**